cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -162,6 +163,19 @@ func (spm *KubeStaticPodPathManager) CleanupDirs() error {
 	return nil
 }
 
+func filesEqual(path1, path2 string) (bool, error) {
```

Comment:
i couldn't find a utility for this in k8s and kubeadm.

we should probably move it here for now:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/staticpod/utils.go
and add a unit test.

possibly call it `ManifestFilesAreEqual`?